packages/calcite-components/src/components/list-item/list-item.scss

```diff
@@ -206,6 +211,40 @@ td:focus {
   }
 }
 
+tr:focus {
+  .actions-start,
+  .actions-end {
+    inset-block: theme("spacing[0.5]");
```

Comment:
note: At some point these theme() vars will need to be updated to use calcite css vars.